morphia/src/main/java/org/mongodb/morphia/query/Query.java

```diff
@@ -15,166 +15,142 @@
  * @author Scott Hernandez
```

Comment:
I mentioned in the PR comments that this file was simply reorganized and could be ignored.